cea/optimization/optimization_main.py

```diff
@@ -147,10 +150,10 @@ def check_input_files(config, locator):
         raise ValueError(
             "Missing sewage potential of the scenario. Consider running sewage heat exchanger script first.")
     if not os.path.exists(locator.get_lake_potential()):
-        raise ValueError("Missing lake potential of the scenario. Consider running lake potential script first.")
+        raise ValueError("Missing water-body potential of the scenario. Consider running lake potential script first.")
```

Comment:
"Consider running _lake potential_ script first"... I think only half of that error message was updated. Additionally, I think the locator method (`get_lake_potential`) could also be updated to `get_water_body_potential`.

Reply:
done!